CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ set_property(TARGET ${PROJECT_NAME} PROPERTY C_STANDARD 99)
 # Required dependencies
 pkg_check_modules(REQ_LIBS REQUIRED libudev libmodule>=5.0.0 libjpeg)
 pkg_check_modules(POLKIT REQUIRED polkit-gobject-1)
+pkg_check_modules(DBUS REQUIRED dbus-1)
+if(WITH_DDC)
```

Comment:
Does ddcutil require systemd? It does not seem so, looking at their configure.ac: https://github.com/rockowitz/ddcutil/blob/master/configure.ac.
DBUS check looks fine, thanks!

Reply:
> Does ddcutil require systemd?

See https://github.com/FedeDP/Clightd/blob/master/CMakeLists.txt#L144

Reply:
I see, thanks, didn't notice!